add bond use case

Bookcliff opened the pull request:
<img width="482" alt="image" src="https://user-images.githubusercontent.com/99197390/208138656-3580f2f5-2e71-4c51-8932-c92ab5edfa83.png">

Namaskar-1F64F commented:
This won't work with the promissory note on mainnet as there are already 6 things in the information. I think it would be better to put under the description 

as a subheading here, 


# Shapeshift DAO information
Shapeshift DAO is a borderless, cross-chain crypto trading platform and portfolio manager enabling user sovereignty.
## Bond use
blah blah

Bookcliff commented:
Fixed

<img width="602" alt="image" src="https://user-images.githubusercontent.com/99197390/208146567-e07c188d-4a70-4422-a203-cf5238f1f331.png">

Namaskar-1F64F commented:
Bond use -> Use of funds

remove the lines

make _here_ blue. text-[#6CADFB]

Bookcliff commented:
> Bond use -> Use of funds
> 
> remove the lines
> 
> make _here_
> 
> * blue. `text-[#6CADFB]`
> * not italic

Fixed

<img width="620" alt="image" src="https://user-images.githubusercontent.com/99197390/208177940-f099da93-05f7-4fbc-b176-e76aa8c8cf8e.png">

Namaskar-1F64F commented on src/pages/BondDetail/index.tsx:
```diff
@@ -403,20 +404,19 @@ const BondDetail: React.FC = () => {
+                            rel="noreferrer"
+                            target="_blank"
+                          >
+                            &nbsp;here.
```
the `&nbsp;` would be fine, but you already have the space after it `{' '}`. Since we are using that in other places, I would prefer sticking with the `{' '}` and removing the `&nbsp;`

Bookcliff replied:
The {' '} doesn't format it to have a space correctly. 
<img width="355" alt="image" src="https://user-images.githubusercontent.com/99197390/208182125-73f13a7a-600b-4789-9296-39415459e809.png">

I think it's because the <a> tag is pushed on the second line, but the &nbsp; does work.

Bookcliff commented:
Removed lines and fixed spacing so text goes with headers respectively

<img width="741" alt="image" src="https://user-images.githubusercontent.com/99197390/208461074-fd73f2c6-dee7-46e0-b3c6-39ed900933e7.png">

Namaskar-1F64F commented:
lgtm!
